dev/start.sh

```diff
@@ -1,4 +1,3 @@
-git checkout develop
-git pull
-#sed -i "s|%COMMIT_HASH%|$(git rev-parse --short HEAD)|g" /ui/src/app/features/footer/footer.component.html
-docker-compose -f docker-compose.yml up --build --scale compreface-postgres-db=0
+#sed -i "s|%COMMIT_HASH%|$(git rev-parse --short HEAD)|g" ui/src/app/features/footer/footer.component.html
+docker-compose -f docker-compose.yml -f docker-compose.dev.ui.yml up --build &
```

Comment:
just to check this config is working, did you try to start with this config on a fresh docker? (after docker system prune)

Reply:
Yes, this config is working

Reply:
once we check that steps in dev readme also work on windows, start.sh can also be removed